docs/nodes/validate/how-to-stake.md

```diff
@@ -78,6 +78,7 @@ probability that a given validator is sampled is proportional to its stake.
 When you add a node to the validator set, you specify:
 
 - Your node’s ID
+- Your node's BLS key and BLS signature
 - When you want to start and stop validating
```

Comment:
you can't specify when to start

Reply:
also, you don't specify when delegator starts (`Delegators` section)

Reply:
also, under `Error: Couldn't Issue TX: Validator Would Be Over Delegated` section, there is no starttime check anymore